src/common/main.lua

```diff
@@ -324,6 +324,34 @@ function ant.init()
 			Names = msg.Tags.Names,
 		})
 	end)
+
+	Handlers.prepend("_bootAnt", function(msg)
+		return msg.Tags.Type == "Process" and Owner == msg.From
+	end, function(msg)
+		if msg.Data and msg.Tags["Initialize-State"] then
+			local status, res = xpcall(function()
+				initialize.initializeANTState(msg.Data)
+			end, utils.errorHandler)
+			if not status then
+				ao.send(notices.addForwardedTags(msg, {
+					Target = Owner,
+					Error = res or "",
+					Data = res or "",
+					Action = "Invalid-Boot-Ant-Notice",
```

Comment:
This will change to `Invalid-Boot-Notice`

Reply:
https://github.com/ar-io/ar-io-ant-process/pull/57/commits/7f04c36b536011cec5286215a0ef1bd830c54ee2